core/src/main/java/org/elasticsearch/search/aggregations/bucket/range/date/DateRangeAggregationBuilder.java

```diff
@@ -259,6 +259,7 @@ public DateRangeAggregationBuilder addUnboundedFrom(DateTime from) {
     @Override
     protected DateRangeAggregatorFactory innerBuild(AggregationContext context, ValuesSourceConfig<Numeric> config,
             AggregatorFactory<?> parent, Builder subFactoriesBuilder) throws IOException {
+        Range[] ranges = processRanges(context, config);
```

Comment:
can we add an inline comment to make it clear why we process anything here. ie. explain why we added it